src/VirtoCommerce.Platform.Web/Security/ApplicationBuilderExtensions.cs

```diff
@@ -39,7 +39,8 @@ public static async Task<IApplicationBuilder> UseDefaultUsersAsync(this IApplica
                         IsAdministrator = true,
                         UserName = "admin",
                         PasswordHash = "AHQSmKnSLYrzj9vtdDWWnUXojjpmuDW2cHvWloGL9UL3TC9UCfBmbIuR2YCyg4BpNg==",
-                        PasswordExpired = true
+                        PasswordExpired = true,
+                        Email = "admin@vc-demostore.com"
```

Comment:
Could you please add NormalizedEmail in uppercase here to not have it empty, like for all other users:
![image](https://user-images.githubusercontent.com/15610352/79998345-03359d80-84d4-11ea-8a37-171469051673.png)

Reply:
Field 'NormalizedEmail' updates automatically when we set field 'email'